README.md

```diff
@@ -2,6 +2,25 @@
 
 # Distributed Linear Algebra from the Future
 
+DLA-Future is a distributed linear algebra library implemented using C++ proposal P2300 which provides:
+- An asynchronous C++ interface,
+- A synchronous C interface,
+- A synchronous ScaLAPACK-like C interface (ScaLAPACK drop-in with a subset of ScaLAPACK arguments. E.g. workspace arguments are not present),
+- A synchronous ScaLAPACK-like Fortran interface (See [DLA-Future-Fortran](https://github.com/eth-cscs/DLA-Future-Fortran)).
+
```

Comment:
Maybe here, or up in the first sentence:
`DLA-Future runs on x86 and ARM CPUs as well as NVIDIA and AMD GPUs.`?